src/components/rocket/Rocket.module.css

```diff
@@ -0,0 +1,49 @@
+.container {
+  display: flex;
+  margin-top: 20px;
+}
+
+.cardImg {
+  width: 450px;
+  height: 250px;
+  margin: 0 20px;
+}
+
+.textContainer {
+  display: flex;
+  flex-direction: column;
+  margin-right: 40px;
+}
+
+.cardTitle {
+  font-size: 30px;
+  font-weight: bolder;
+}
+
+.cardText {
+  padding: 20px 0;
+  line-height: 1.7;
+  font-size: 20px;
+}
+
+.buttonName {
+  background-color: aqua;
+  height: 20px;
+  width: 40px;
+  margin-right: 10px;
+}
+
+.cancelBtn {
+  display: block;
+  width: 150px;
+  height: 40px;
+}
+
+.reserveBtn {
+  display: block;
+  width: 120px;
+  height: 35px;
+  background-color: #318ce7;
+  color: white;
+  border: none;
+}
```

Comment:
- [ ] Good job so far but I think you should work a little bit on your buttons design, you should make sure your buttons stick to the given design, the colors, and the border-radius 🙃 😉 

Your design

<img width="974" alt="Screenshot 2023-04-14 at 01 02 02" src="https://user-images.githubusercontent.com/74501523/231894773-9c007364-2a25-472e-9a2d-52194436668a.png">

Given design

<img width="1181" alt="space-travelers__rockets" src="https://user-images.githubusercontent.com/74501523/231894945-73d69420-f477-4c93-a976-ff8393143e1f.png">